library/stdtypes.po

```diff
@@ -2816,20 +2821,18 @@ msgstr ""
 "est introuvable."
 
 #: ../Doc/library/stdtypes.rst:1665
-#, fuzzy
 msgid ""
 "Return ``True`` if all characters in the string are alphanumeric and there "
 "is at least one character, ``False`` otherwise.  A character ``c`` is "
 "alphanumeric if one of the following returns ``True``: ``c.isalpha()``, ``c."
 "isdecimal()``, ``c.isdigit()``, or ``c.isnumeric()``."
 msgstr ""
-"Donne ``True`` si tous les caractères de la chaîne sont alphanumériques et "
+"Renvoie ``True`` si tous les caractères de la chaîne sont alphanumériques et "
 "qu'il y a au moins un caractère, sinon ``False``. Un caractère ``c`` est "
-"alphanumérique si l'un des tests suivants est vrais : ``c.isalpha()``, ``c."
-"isdecimal()``, ``c.isdigit()`` ou ``c.isnumeric()``."
+"alphanumérique si l'un des tests suivants renvoie ``True`` : ``c."
```

Comment:
```suggestion
"alphanumérique si l'un des tests suivants renvoie ``True`` : ``c."
```

/tmp/padpo_j77_iede/library/stdtypes.po:2823 [     NBSP     ] There should be a non-breakable space before ":": between ###ests suivants renvoie « True » ### and ###: « c.isalpha() », « c.isdecima###